client/src/components/DateTime/__tests__/DateSelect.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import DateSelect from '../DateSelect';
+
+const noop = () => {};
+
+test('renders correctly', () => {
```

Comment:
if these are all just doing render/snapshot tests you should probably just put them all in the same `test()` like I did for the others I created - this seemed to be the recommended way from the jest docs.